### PR TITLE
Add projection aware Graph and TriMesh elements

### PR DESCRIPTION
**Pull request by philippjfr**
This adds support for projection aware ``Graph`` elements to GeoViews including the basic ``Graph`` element and the ``TriMesh`` element. It relies on https://github.com/ioam/holoviews/pull/2240 to work correctly.

### CODE REVIEW

**Comment by jbednar** [comment]
Looks good to me!

File: geoviews/operation.py [hunk 1]
```diff
@@ -90,23 +88,27 @@ def _process_element(self, element):
...
+
+class project_graph(_project_operation):
+
+    supported_types = [Graph]
+
+    def _process_element(self, element):
+        nodes = project_points(element.nodes, projection=self.projection)
+        data = (element.data, nodes)
+        if element._edgepaths:
+            data = data + (project_path(element.edgepaths),)
```
Comment (philippjfr) [inline geoviews/operation.py]: Forgot passing the ``projection`` here.

**Comment by philippjfr** [comment]
Almost ready, needs pyflakes fixes, need to define style/plot options for the new elements and one small fix I noted above. Also would be good to merge https://github.com/ioam/holoviews/pull/2240 first.

**Comment by philippjfr** [comment]
The HoloViews PR is now merged and this is working as expected. Will go ahead and merge.

**Pull request merged**